float64 goalLatitude
float64 goalLongitude
---
geometry_msgs/Point[] pathCoordinates

